src/main/resources/reference.conf

```diff
@@ -826,11 +831,17 @@ resourceTypes = {
     }
     ownerRoleName = "creator"
     roles = {
+      # the creator role is assigned directly to the notebook-cluster resource and does not include can_access
+      # but can_access is required to perform any action so must be granted via the parent resource using
+      # either the manager or accessor role
       creator = {
         roleActions = ["status", "connect", "delete", "read_policies", "stop_start", "modify", "set_parent", "get_parent"]
       }
       manager = {
-        roleActions = ["status", "delete", "read_policies"]
+        roleActions = ["status", "delete", "read_policies", "can_access"]
+      }
```

Comment:
I see you added `descendantRoles` for `accessor` but not `manager` (for both notebook-cluster and kubernetes-app). How is `manager` assigned to users? Is that assigned directly somewhere by Leo?

Reply:
manager is already a descendant role from workspace owner